packages/addons/addon-depends/vdr-plugins/vdr-plugin-vnsiserver/package.mk

```diff
@@ -9,6 +9,7 @@ PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/mdre77/vdr-plugin-vnsiserver"
 PKG_URL="https://github.com/mdre77/vdr-plugin-vnsiserver/archive/${PKG_VERSION}.tar.gz"
 PKG_DEPENDS_TARGET="toolchain vdr"
+PKG_DEPENDS_UNPACK="vdr-plugin-wirbelscan"
```

Comment:
`PKG_DEPENDS_UNPACK=vdr-plugin-wirbelscan` is an alias for `PKG_NEED_UNPACK=$(get_pkg_directory vdr-plugin-wirbelscan)`  which already exist.

But we do need `PKG_DEPENDS_TARGET="toolchain vdr vdr-plugin-wirbelscan"`

Reply:
The build was regularly erroring without `PKG_DEPENDS_UNPACK="vdr-plugin-wirbelscan"` - so I am unsure? I was able to reproduce the failure from a clea build without `PKG_DEPENDS_UNPACK="vdr-plugin-wirbelscan"`

Reply:
[The code](https://github.com/LibreELEC/LibreELEC.tv/blob/144a8cd397e357c089e5e6b8327d469f2712d129/config/functions#L1280-L1284) is definite, you may have just won the race.

For build dependencies we have to use PKG_DEPENDS_TARGET.